src/components/FormComponents/Pasientopplysninger/Adresser/Utenlandskadresse.tsx

```diff
@@ -0,0 +1,24 @@
+import { BodyShort } from '@navikt/ds-react';
+
+import { UtenlandskAdresse } from '../../../../graphql/queries/graphql.generated';
+import { checkValue } from '../../../../utils/formUtils';
+import countries from '../../../../pages/api/country/countries-norwegian.json';
```

Comment:
Om vi importerer disse sånn direkte i frontend-koden så vil de bli automatisk bundlet inn i bundlen.

Det er ikke verdens undergang det altså, men da kan vi forenkle CountryCombobox ganske mye og fjerne det fetch-kallet som gjøres der for å hente landene asynkront. Alternativt kan du velge å heller bruke `useCountrySuggestions` her, så blir denne funksjonen også asynkron og henter countries on-demand.